src/robot/libraries/String.py

```diff
@@ -606,15 +682,25 @@ def split_string_from_right(self, string, separator=None, max_split=-1):
         max_split = self._convert_to_integer(max_split, "max_split")
         return string.rsplit(separator, max_split)
 
-    def split_string_to_characters(self, string):
+    # TODO(@pekkaklarck):
+    # def split_string_to_characters(self, string: "str | bytes") -> list["str | bytes"]:
+    # does not work:  Split String To Characters    ${{b'ab 12'}} -> [97, 98, 32, 49, 50]   int represtantion in robot hmmm
```

Comment:
This is due to `bytes` being considered a sequence of integers in Python:
```python
>>> list(b'hello')
[104, 101, 108, 108, 111] 
```
I guess it would make sense in the context of this keyword to convert those integers to byte characters. Can be done like this:
```python
>>> [bytes([i]) for i in list(b'hello')]
[b'h', b'e', b'l', b'l', b'o']
```
Doing that as part of this issue doesn't make sense so better to just mention this in #5548. The initial typing could nevertheless be `str | bytes`.